### PR TITLE
[Rec-IM] remove co captain

### PR DESCRIPTION
**Pull request by antoniavonto**
Remove promote to co-captain.
Implemented promote to captain, if you promote to captain, it sets the previous captain back to member.
image

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # VITE_API_URL=https://360Api.gordon.edu/
 
 # @TRAIN
- VITE_API_URL=https://360ApiTrain.gordon.edu/
+ VITE_API_URL=https://360apirecim.gordon.edu/
```
Comment (bennettforkner) [inline .env.development:7]: Please make sure to revert this change before merging into recim or develop.

File: src/views/RecIM/components/List/index.jsx [hunk 2]
```diff
@@ -42,6 +42,28 @@ const ParticipantList = ({
...
   callbackFunction,
 }) => {
+  const [newCaptain, setNewCaptain] = useState();
+  const [teamId, setTeamID] = useState();
+
+  useEffect(() => {
+    // find the old capitain to demote
+    participants.map(async (participant) => {
+      if (
+        participants.length > 0 &&
```
Comment (bennettforkner) [inline src/views/RecIM/components/List/index.jsx:52]: If you're inside this lambda function, you already know that participants.length is at least 1.
```suggestion
```

File: src/views/RecIM/components/List/index.jsx [hunk 3]
```diff
@@ -42,6 +42,28 @@ const ParticipantList = ({
...
+        participant !== newCaptain &&
+        teamId !== undefined
+      ) {
+        let editedParticipant = {
+          Username: participant.Username,
+          RoleTypeID: 3,
+        }; // Role 3 is member
+        await editTeamParticipant(teamId, editedParticipant);
+      }
+      callbackFunction((bool) => !bool);
```
Comment (bennettforkner) [inline src/views/RecIM/components/List/index.jsx:63]: I wouldn't name a variable after a common type, `boolean`. That will confuse other developers. It would be better to do something simple like a character.
```suggestion
      callbackFunction((b) => !b);
```

File: src/views/RecIM/components/List/index.jsx [hunk 4]
```diff
@@ -64,8 +88,8 @@ const ParticipantList = ({
         teamID={teamID}
         callbackFunction={(bool) => callbackFunction(bool)}
         showParticipantOptions={
-          showParticipantOptions &&
-          participant.Role !== 'Team-captain/Creator' &&
+          showParticipantOptions ||
```
Comment (antoniavonto) [inline src/views/RecIM/components/List/index.jsx:91]: showParticipantOptions was returning false so then the buttons to promote to captain and remove from team were not showing for me. It worked when I changed to "||", however I am not sure if it would be wrong for other parts of the code.

**Review by amos-cha — CHANGES_REQUESTED** [review]
Good work! Just minor things then it should be ready to go :)
